docs/source/contents/getting-started/kubernetes-installation/helm.md

```diff
@@ -7,17 +7,24 @@ We provide several Helm charts.
  * `seldon-core-v2-servers` : a default set of servers
  * `seldon-core-v2-certs` : a default set of certificates for TLS
 
-The Helm charts can be found within the `k8s/helm-charts` folder.
+The Helm charts can be found within the `k8s/helm-charts` folder and they are published [here](https://github.com/SeldonIO/helm-charts)
 
 Assuming you have installed any ecosystem components: Jaeger, Prometheus, Kafka as discussed [here](./index.md) you can follow the
 following steps.
 
-The Helm charts are provided as part of the repository. To use the instructions below clone the repostitory. They will be published in future.
+Note that for Kafka follow the steps discussed [here](kafka.md) 
```

Comment:
Would be good to also advice that one can connect to existing / managed Kafka + link / discuss relevant config options.